.ktlintignore

```diff
@@ -0,0 +1,7 @@
+// disabled ("./mvnw clean verify" fails with "Internal Error")
```

Comment:
nit: .xyzignore files are conventionally lists of file expressions to ignore. What if it was `.ktlintconfig` and perhaps in something like toml?

Reply:
Yes - it's goofy. Probably a better format is something like `.editorconfig`:

```
[*]
disable_rules=no-wildcard-imports

[**/library/*]
disable_rules=no-wildcard-imports,package-name
```

Reply:
> Yes - it's goofy. Probably a better format is something like .editorconfig:

yes, this format of the file looks better and will allow in the future add more `ktlint` specific configuration here.